(execute) - Fix imports and reuse getOperationName from core (#1135)
FormidableLabs/urql: .changeset/popular-bananas-camp.md

---
"@urql/core": patch
"@urql/exchange-execute": patch
---

> add the getOperationName export to @urql/core and use the newly exported getOperationName in @urql/exchange-execute.

Reviewer: ```suggestion
Export `getOperationName` from `@urql/core` and use it in `@urql/exchange-execute`, fixing several imports.
```
